tutorials/plot_sensors_time_frequency.py

```diff
@@ -70,12 +72,103 @@
 
 ax.plot(freqs, psds_mean, color='k')
 ax.fill_between(freqs, psds_mean - psds_std, psds_mean + psds_std,
-                color='k', alpha=.5)
+                color='k', alpha=.15, lw=0)
 ax.set(title='Multitaper PSD (gradiometers)', xlabel='Frequency',
        ylabel='Power Spectral Density (dB)')
 plt.show()
 
 ###############################################################################
+# To see the difference between :func:`psd_multitaper` and :func:`psd_welch`
+# - we are going to compare PSDs obtained with each function.
+ch_index = epochs.ch_names.index('MEG 2333')
+
+psds_m, freqs_m = psd_multitaper(epochs, picks=[ch_index],
+                                 fmin=2, fmax=17, n_jobs=1)
+psds_w, freqs_w = psd_welch(epochs, n_fft=epochs.info['sfreq'] * 2,
+                            picks=[ch_index], fmin=2, fmax=17, n_jobs=1)
+
+# drop channel dimension and average epochs
+psd_m = psds_m.squeeze(axis=1).mean(axis=0)
```

Comment:
Ok, I can use `[:, 0]` instead - but can squeeze be dangerous if I specify the axis?